src/app/form-elements/radio/radio-group-sample/radio-group-sample.component.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <label>Choose a fruit and submit your choice:</label>
+    <form [formGroup]="fruitsForm" (ngSubmit)="onSubmit()">
+        <igx-input-group>
+            <input igxInput name="fullName" type="text" formControlName="fullName" required />
+            <label igxLabel for="fullName">Full Name</label>
+        </igx-input-group>
+
+        <igx-radio-group name="fruitsRadioGroup" formControlName="favoriteFruit">
+            <igx-radio class="radio-sample" *ngFor="let fruit of fruits" value="{{fruit}}">
+                {{fruit}}
+            </igx-radio>
+        </igx-radio-group>
+        <button igxButton="raised" igxRipple type="submit">Submit</button>
```

Comment:
Besides the submit button please add a reset button. It should reset the values in the form.